app/main/views/users.py

```diff
@@ -296,3 +296,32 @@ def check_supplier_role(role, supplier_id):
         abort(400, "'supplier_id' is required for users with 'supplier' role")
     elif role != 'supplier' and supplier_id:
         abort(400, "'supplier_id' is only valid for users with 'supplier' role, not '{}'".format(role))
+
+
+@main.route("/buyers/briefs", methods=["GET"])
```

Comment:
While this gives us exactly what we need, I feel like this might be a bit too standalone for the API. It's a new endpoint and a response type that's not tied to a model and unlikely to be used by anything else.

What if we extend `/briefs` with an additional GET parameter to inline user data instead of user IDs? This gives the admin app all the data needed to build the list of buyers with associated briefs. We'd still need to paginate through the briefs and the API responses will contain all brief data that we don't really need, but as long as it's not timing out efficiency doesn't seem like a crucial concern for this.

Alternatively, if we also want to include buyer users that don't have any briefs we could add a `role` filter to `/users` and make admin app request all briefs and all buyer users and join them in place. If that proves to be too slow we could look into adding a way to request a full users/brief list without pagination, which should cut down the number of requests. The main benefit (and I think it's an important one) compared to a custom endpoint then would be that these could be reused by other scripts/tasks.